common/testData/localeIdentifiers/localeCanonicalization.txt

```diff
@@ -307,7 +308,7 @@ nau	;	na
 nav	;	nv
 nbf	;	nru
 nbl	;	nr
-nbx	;	ekc
+nbx	;	gll
```

Comment:
ekc = Eastern Karnic and nbx = Ngura. [The ISO 639 website](https://iso639-3.sil.org/code/ekc) for either language, points to [the same wikipedia page](https://en.wikipedia.org/wiki/Ngura_languages) which points out that this language is not well-attested.
>>> Ngura is a disputed and possibly spurious ethnic and language designation of central Australia. The name 'Nura', short for Ngurawarla, means 'empty camp', referring to lands abandoned after a massacre. It is not a language or ethnic designation

gll = Kalali, https://en.wikipedia.org/wiki/Kalali_language